src/lessons/11_dom_part2/dom_part2.slides.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Javascript: Dom manipulations part 2</title>
+  <meta name="date" content="2021-04-26" />
+  <meta name="description" content="
+      - Recap of DOM manipulations
+      - Introduction on manipulation of styles
+    " />
+</head>
+
+<body>
+  <div class="reveal">
+    <div class="slides" id="javascript-recap-dom">
+      <section>
+        <p>Hey! 👋🏼</p>
+        <h1 class="r-fit-text">Javascript: DOM Manipulation Pt.2</h1>
+        <p>Redi School Munich - Spring 2021</p>
+      </section>
+
+      <section class="initial">
+        <section data-auto-animate data-timing="10">
+          <h2>Recap</h2>
+        </section>
+
+        <section>
+          <h2>Document Object Model</h2>
+          <p>Javascript's interface for interacting with HTML</p>
+          <small>MDN:
+            <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction">Document
+              Object Model</a></small>
+          <ul>
+            <li class="fragment">getElementById()<br>
+              <code class="fragment formattedCode" data-trim>document.getElementById('main')</code>
+            </li>
+            <li class="fragment">querySelector()<br>
+              <code class="fragment formattedCode" data-trim>document.querySelector('li')</code>
+              <code class="fragment formattedCode" data-trim>document.querySelector('.someclass')</code>
+            </li>
+            <li class="fragment">innerHTML<br>
+              <code class="fragment formattedCode" data-trim>document.querySelector('li').innerHTML = 'Email <a
+                  href="mom@gmail.com">Mom</a>.';;</code>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>Some more on DOM</h2>
+          <ul>
+            <li class="fragment">textContent<br>
+              <code class="fragment formattedCode" data-trim>document.querySelector('li').textContent</code>
+            </li>
+            <li class="fragment">createElement()<br>
+              <code class="fragment formattedCode" data-trim>const newParagraph = document.createElement('p')</code>
+            </li>
+            <li class="fragment">appendChild()<br>
+              <code class="fragment formattedCode" data-trim>firstHeading.appendChild(newParagraph)</code>
+            </li>
+          </ul>
+        </section>
+
+        <section>
+          <h3>In-class exercise!</h3>
+
+          <p><small>1.5 hours!</small> 🕘</p>
+
+          <ul>
+            <li>
+              Display 10 movie titles according to an array of nominees from this years Oscar!
+            </li>
```

Comment:
Btw, here's what I thought of for this: (just created 😛 )

```javascript
const movies = [
    { title: "Judas and the Black Messiah", stars: 5, url: "#" },
    { title: "Ma Rainey’s Black Bottom", stars: 4, url: "#" },
    { title: "Mank", stars: 4, url: "#" },
    { title: "Minari", stars: 4, url: "#" },
    { title: "Nomadland", stars: 4, url: "#" },
    { title: "Promising Young Woman", stars: 3, url: "#" },
    { title: "Sound of Metal", stars: 3, url: "#" },
    { title: "The Father", stars: 3, url: "#" },
    { title: "The Trial of the Chicago 7", stars: 3, url: "#" },
    { title: "The United States vs. Billie Holiday", stars: 3, url: "#" },
]
```

Reply:
ah great, then we will use this :)